.github/workflows/nix.yml

```diff
@@ -16,7 +16,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        task: [ "coq-8.13", "coq-8.12", "coq-8.11" ]
+        bundle: [ "coq-8.13", "coq-8.12", "coq-8.11" ]
```

Comment:
Is this new?

Reply:
This is a renaming... we are converging to v1.0.0 of coq-nix-toolbox with Théo... we are very close to the release now...

Reply:
after that no more change...

Reply:
(there is a `format` variable in `.nix/config.nix` to allow for future updates)

Reply:
I mean, the bundle keyword here, I could not find it in the doc of GH actions. What does it do?

Reply:
nor was task... this was always a bound variable

Reply:
yml has bound variables?

Reply:
oh, I see. thanks for clarifying